src/components/Popover/docs/Popover.mdx

```diff
@@ -0,0 +1,125 @@
+---
+name: Popover
+menu: Components
+route: /components/Popover
+---
+
+import { Playground } from 'docz';
+import { ItemWrapper } from '../../../../docs/components/ItemWrapper.ts';
+import { SettingsIcon } from '../../../icons/index';
+
+import { Button } from '../../Button/Button';
+import { Text } from '../../Text/Text';
+import { Link } from '../../Link/Link';
+
+import { Popover } from '../Popover.tsx';
+
+import { PopoverPropsTable } from './PopoverPropsTable.tsx';
+
+
+# Popover
+
+**Primary UI element for content whose visibility can be toggled with a click on trigger element**
+
+Renders a UI component that opens a popup dialog on click of a trigger element.
```

Comment:
```suggestion
Renders a UI component that opens a popup dialog when clicking on a trigger element.
```

Reply:
I guess, I would then change to ''_**when user clicks on** a trigger element_'', if we want to be more specific 🤔